zooniverse/front-end-monorepo
# lib-user: add user_group actions, related components for local dev

## Package
- lib-user

## Linked Issue and/or Talk Post
- towards user stats and group stats efforts

## Describe your changes
- add user_group actions for create, update, and delete
- add initial MyGroups component

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What user actions should my reviewer step through to review this PR?
  1. from `lib-user/` run `yarn dev`
  2. go to `https://local.zooniverse.org:8080/?users=[your staging user login here]/groups`
  3. note your groups, if you have any
  4. create a group (at the bottom)
  5. go to your new group at `https://local.zooniverse.org:8080/?groups=[new group id here]`
  6. edit your group display_name (towards the bottom)
  7. (optional) delete your group (towards the bottom)
- Which storybook stories should be reviewed? none yet
- Are there plans for follow up PR’s to further fix this bug or develop this feature? yes, see linked GitHub project FEM Stats Efforts
